### ISSUE TITLE
WebStorm 2023.3 EAP support

### ISSUE THREAD

**Issue by niklv** [one-dark/jetbrains-one-dark-theme#314]
![изображение](https://github.com/one-dark/jetbrains-one-dark-theme/assets/3831943/1ff7fb5c-6332-4c60-b760-963999237e83)

**Comment by danwilliams** [comment]
@simPod Just to say, I get the same error with RustRover 2023.3, so I downgraded... mentioning it here just in case it's not covered by the fix for WebStorm. I didn't want to create another ticket as this one's basically the same thing :smile:

**Comment by simPod** [comment]
You can use my plugin build from the PR

**Comment by danwilliams** [comment]
@simPod thanks, that's okay - I'll wait for the official release, but I just wanted to comment to ensure you knew, and that RustRover was covered :slightly_smiling_face:

**Comment by simPod** [comment]
I'm not a maintainer :P

**Comment by crypt0grapher** [comment]
experiencing the same! love the team so much that I was thinking to downgrade Rust Rover to use it :)
This custom build from the PR works like a charm for now!
https://github.com/one-dark/jetbrains-one-dark-theme/pull/315#issuecomment-1748932277

**Comment by danwilliams** [comment]
@simPod oh, sorry!

Well, to whoever _is_ a maintainer - I upgraded just now, based on the new release dropping, which was just in time as the previous RustRover build just expired so it was forcing me to upgrade anyway.

OneDark is compatible again - but weirdly, the "One Dark Italic" theme now has a white background with dark text. Meanwhile the One Dark Vivid Italic still works, so I've changed to that for now.

Hmmm actually, I should probably open a ticket about this...
